project/currency_comparison/http.js

```diff
@@ -0,0 +1,9 @@
+const axios = require('axios')
+
+const httpRequest = async () => {
+    const rates = await axios.get('https://api.ratesapi.io/api/latest')
+    rates.status && console.log(`REST API status: ${rates.status}`)
+    return rates.data
+}
```

Comment:
Since this is the function that makes the axios call, we should have a `http.test.js` file that uses the mock to test this functionality.